### PR TITLE
Expose notebook culling parameters for configuration

### PR DESCRIPTION
**Pull request by jsitu777**
**Which issue is resolved by this Pull Request:**
Resolves # https://github.com/awslabs/kubeflow-manifests/issues/439

**Description of your changes:**
* create overlay to expose culling parameters for notebook controller
* change related notebook paths

**Testing:**
- [ ] Unit tests pass
- [x] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsconfigs/apps/notebook-controller/deployment_patch.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: notebook-controller-deployment
+  namespace: kubeflow
+spec:
+  template:
+    spec:
+      containers:
+      - command:
```
Comment (surajkota) [inline awsconfigs/apps/notebook-controller/deployment_patch.yaml:10]: I dont think you anything except the name of container, can you check if specifying name of container is sufficient?

File: charts/apps/notebook-controller/values.yaml [hunk 2]
```diff
@@ -1,2 +1,3 @@
-null
-...
+cullIdleTime: 30
```
Comment (surajkota) [inline charts/apps/notebook-controller/values.yaml:1]: update this to

```
cullingPolicy:
  enableCulling:
  cullIdleTime:
  idlenessCheckPeriod:
```

and make corresponding chart changes

**Review by surajkota — CHANGES_REQUESTED** [review]
Thanks for investigation. Looks good with minor comments. Lets make the small changes needed for this PR and merge it

Iterate on terraform and doc changes separately

File: awsconfigs/apps/notebook-controller/deployment_patch.yaml (hunk 1 above)
Comment (jsitu777) [reply awsconfigs/apps/notebook-controller/deployment_patch.yaml]: fixed.

File: charts/apps/notebook-controller/values.yaml (hunk 2 above)
Comment (jsitu777) [reply charts/apps/notebook-controller/values.yaml]: done

**Review by surajkota — APPROVED** [review]

**Pull request merged**